packages/slice-machine/pages/slices.tsx

```diff
@@ -94,10 +93,33 @@ const SlicesIndex = ({ env }: { env: Environment }) => {
   }, []);
   const hasConfigLocalLibs = configLocalLibs.length;
 
+  const sliceCount =
+    libraries && libraries.length
```

Comment:
Judging by the localLibs deifnition, libraries doesn't seem to be undefined, I would just push the reduce for slice count, no need to verify it's length.

Reply:
Nop the type is LibrairyState | undefined, this is because the injected payload type mark the "libraries" key as optional